test/support/data_case.ex

```diff
@@ -51,4 +53,28 @@ defmodule Cadet.DataCase do
       end)
     end)
   end
+
+  @doc """
+  Run a git command with `loc` as the git repository root.
+  """
+  def git_from(loc, cmd, opts \\ []) do
```

Comment:
If the following functions are used only by the deployment test, I suggest moving this either to the test file directly as a `defp` or a separate file `updater_helper.exs`

Reply:
Fixed with 7874796.